lib/jxl/decode.cc

```diff
@@ -2653,28 +2662,63 @@ JxlDecoderStatus JxlDecoderGetFrameName(const JxlDecoder* dec, char* name,
 
 JxlDecoderStatus JxlDecoderSetPreferredColorProfile(
     JxlDecoder* dec, const JxlColorEncoding* color_encoding) {
+  return JxlDecoderSetOutputColorProfile(dec, color_encoding, nullptr, 0);
```

Comment:
Parameter comments for the last two arguments would be helpful.

Reply:
done